src/Raven.Studio/wwwroot/Content/css/icons.less

```diff
@@ -160,6 +160,7 @@
 .make-icon(lets-encrypt, "\e9df");
 .make-icon(chash, "\e9dd");
 .make-icon(minus, "\e9dc");
+.make-icon(average, "\e9dc");
```

Comment:
Did you include other icon files in this PR? Or average icon was already present?

Reply:
It was already present, I've reused other icon but with different class name.

Reply:
oh, I can see it uses the same code as icon-minus. Thanks